### ISSUE TITLE
how can i get city name and cordinates of each tweet?

### ISSUE THREAD

**Issue by afzal646** [Mottl/GetOldTweets3#42]
Update to the latest version of GetOldTweets3 before committing the issue!

**Describe the bug**

**debug.log**
Run GetOldTweets with the `--debug` option:
```
GetOldTweets ... --debug > debug.log
```

Upload `debug.log` to somewhere like [http://gist.github.com](gist.github.com) or  [https://pastebin.com](pastern.com) and provide with the link to your `debug.log`.

**For general issues with running GetOldTweets3**
If you have a general question please provide with OS, Python version and the method you have used to install GetOldTweets3

**Comment by Mottl** [comment]
@afzal646, do you see a location info near tweets in a browser?

**Comment by afzal646** [comment]
yes like i this lib using .setnear this tells tweets within given location but how they tell us that these tweets are from this certain portion 
can you tell whats the logic behind setnear(i know it gives tweets from given location but how they extract these tweets whats the logic behind?)
i want to print out city name or coordinates of each tweet?

**Comment by marquisvictor** [comment]
@Mottl do you think it's possible to get city names or state in a specific column?
